ci-operator/step-registry/openshift-extended/test/openshift-extended-test-ref.yaml

```diff
@@ -39,8 +39,8 @@ ref:
       the subsequence step is not executed per https://docs.ci.openshift.org/docs/architecture/step-registry/#workflow
   resources:
     requests:
-      cpu: "3"
-      memory: 600Mi
+      cpu: "1"
+      memory: 1Gi
```

Comment:
The change happens in the `requests` part, it matters, but not the key point, as the `limits` is the one that matters the larges resources you can use.
Even if you require more/ or less of the cpu/memory, the tests will not fail, as OpenShift support allocating resources dynamically.
The update here is to make the resource usage more efficient. E.g, as you can see from the screenshots, we use less than 1 cpu most of the time, default requesting to 3 is a little waste of the resources.